lib/active_record_host_pool/pool_proxy.rb

```diff
@@ -129,7 +129,16 @@ def _connection_pools
     end
 
     def _pool_key
-      @_pool_key ||= "#{@config[:host]}/#{@config[:port]}/#{@config[:socket]}/#{@config[:username]}/#{@config[:slave] && 'slave'}"
+      @_pool_key ||= begin
+        [
+          "#{@config[:host]}",
+          "#{@config[:port]}",
+          "#{@config[:socket]}",
+          "#{@config[:username]}",
+          "#{@config[:slave] && 'slave'}",
+          "#{@config[:database]}"
```

Comment:
no need for all these `"#{}"`

Reply:
Doh!  🤦  I should have just waited til this morning to before making the other change. I've updated this in b6cb625

In doing this I've also swapped the position of the slave market and the database name. I did this since the slave marker may not be present.